dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -63,6 +63,7 @@ def next
   end
 
   def show
+    @current_user = current_user && User.includes(:teachers).where(id: current_user.id).first
```

Comment:
The email I got shows that this PR has changes to 5 files, but github UI only shows this change and the one to `db_query_test`. Is this right?

To make sure I'm properly understanding the query here, the `where` portion is still applied to the user, right? i.e. This is searching for the first user where id is `current_user.id`, and including the result of `user.teachers` in that query.

Reply:
I had written a different implementation originally (involving chaining scope-lambdas against `User.current_scope`, then using an `around_action` controller filter to mix that into the `:show` action), but changed it to this one-liner to make it simpler/safer (scopes are weird and can have surprising bugs/side-effects). I made the change via a force-push to the branch to override the original commit, which is why your original email was different.

And yes, you're understanding the query correctly.